tests/tests/wgpu-validation/api/texture.rs

```diff
@@ -508,3 +508,96 @@ fn copy_buffer_to_texture_forbidden_format_aspect() {
         );
     }
 }
+
+/// Ensures that attempting to create a texture with [`wgpu::TextureUsages::TRANSIENT`]
+/// and its unsupported usages fails validation.
+#[test]
+fn transient_invalid_usage() {
+    let (device, _queue) = wgpu::Device::noop(&wgpu::DeviceDescriptor::default());
+
+    let size = wgpu::Extent3d {
+        width: 256,
+        height: 256,
+        depth_or_array_layers: 1,
+    };
+
+    let invalid_usages = [
+        wgpu::TextureUsages::COPY_SRC,
+        wgpu::TextureUsages::COPY_DST,
+        wgpu::TextureUsages::TEXTURE_BINDING,
+        wgpu::TextureUsages::STORAGE_BINDING,
+        wgpu::TextureUsages::STORAGE_ATOMIC,
+    ];
```

Comment:
I would define this as 

```suggestion
    let invalid_usages = wgpu::TextureUsages::all() - wgpu::TextureUsages::RENDER_ATTACHMENT - wgpu::TextureUsages::TRANSIENT;
```

so that it automatically expands if we add more usages.